.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -17,4 +17,4 @@ assignees: ''
 (e.g. 20.11.1.1)
 
 **Exported settings file**
-(If you think it might be relevant, create a settings export file using the _Export Settings_ button on the settings page; zip or rename the resulting file to .txt to attach to the issue.)
+(If you think it might be relevant, create a settings export file using the _Export Settings_ button on the settings page.)
```

Comment:
Could you keep "and attach it to this issue" so people don't paste it into the ticket body?

Reply:
Fixed.